ReleaseTooling/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -553,6 +548,15 @@ enum CocoaPodUtils {
     return podfile
   }
 
+  private static func isSourcePodspec(_ podspecPath: String) -> Bool {
+    do {
+      let contents = try String(contentsOfFile: podspecPath, encoding: .utf8)
+      return contents.range(of: ".vendored_frameworks") == nil
```

Comment:
optional: if you feel it may be helpful, add a comment explaining why the code checks for the presence of `vendored_frameworks`